Sources/WinUIBackend/WinUIBackend.swift

```diff
@@ -1377,6 +1377,42 @@ public final class WinUIBackend: AppBackend {
         tapGestureTarget.height = tapGestureTarget.child!.height
     }
 
+    public func createHoverTarget(wrapping child: Widget) -> Widget {
+        let hoverTarget = HoverGestureTarget()
+        addChild(child, to: hoverTarget)
+        hoverTarget.child = child
+
+        // Ensure the hover target covers the full area of the child.
+        // Use a transparent background so the visual appearance doesn't change but
+        // the hit-testing covers the whole region.
+        let brush = SolidColorBrush()
+        brush.color = UWP.Color(a: 0, r: 0, g: 0, b: 0)
+        hoverTarget.background = brush
+
+        hoverTarget.pointerEntered.addHandler { [weak hoverTarget] _, _ in
+            guard let hoverTarget else { return }
+            hoverTarget.enterHandler?()
+        }
+        hoverTarget.pointerExited.addHandler { [weak hoverTarget] _, _ in
+            guard let hoverTarget else { return }
+            hoverTarget.exitHandler?()
+        }
+        return hoverTarget
+    }
+
+    public func updateHoverTarget(
+        _ hoverTarget: Widget,
+        environment: EnvironmentValues,
+        action: @escaping (Bool) -> Void
+    ) {
+        let hoverTarget = hoverTarget as! HoverGestureTarget
+        hoverTarget.enterHandler = environment.isEnabled ? { action(true) } : {}
+        hoverTarget.exitHandler = environment.isEnabled ? { action(false) } : {}
+
+        hoverTarget.width = hoverTarget.child!.width
+        hoverTarget.height = hoverTarget.child!.height
```

Comment:
I realise that this was copied from the tap gesture modifier, but I've just noticed that it is probably redundant (in both cases) since both modifiers call `backend.setSize` just before calling the corresponding updateBlahTarget methods.

I'm happy to try that out on my end after merging though if a getting a Windows VM up is a bit annoying on your end.